src/main/resources/application.properties

```diff
@@ -69,6 +69,7 @@ spring.mail.port=
 #Kafka sensor properties.
 kafkaSource.group.id.prefix=hyper_drive
 kafkaSource.poll.duration=500
+kafkaSource.connectionTimeoutMillis=10000
```

Comment:
I think it's problematic to group this together with the configuration for the notification topic. The Kafka service and the kafka sensor share nothing in common except they both connect to some kafka server